container-engine-lib/lib/backend_interface/kurtosis_backend.go

```diff
@@ -31,7 +31,8 @@ type KurtosisBackend interface {
 	// FetchImage always attempts to retrieve the latest image.
 	// If retrieving the latest [dockerImage] fails, the local image will be used.
 	// Returns True is it was retrieved from cloud or False if it's a local image
-	FetchImage(ctx context.Context, image string, downloadMode image_download_mode.ImageDownloadMode) (bool, error)
+	// Returns a string that represents the architecture of the image
```

Comment:
why you removed this comment: ` Returns True is it was retrieved from cloud or False if it's a local image` ?

Reply:
I havne't remvoed it; its still tehre!